app/controllers/api/v1/greetings_controller.rb

```diff
@@ -0,0 +1,52 @@
+class Api::V1::GreetingsController < ApplicationController
+  before_action :set_greeting, only: %i[show update destroy]
+
+  # GET /greetings
+  def index
+    @greetings = Greeting.all
+
+    render json: @greetings
+  end
```

Comment:
- [ ] Good job so far, but there is a little more work to be done. You are expected to return random greetings when this route is hit. At the moment, you're returning all the greetings and so does your frontend. Kindly work on returning random greeting(One greeting) whenever the page is refreshed 👍🏽